Sources/SwiftTUI/Drawing/Cell.swift

```diff
@@ -9,17 +9,21 @@ struct Cell: Equatable {
     /// is used.
     var backgroundColor: Color?
 
+    var highlighted: Bool?
```

Comment:
I'm not a huge fan of this, because it just undoes the "reverted" property, and then it's ignored by the renderer.
Instead of adding this to the cell, I was thinking, maybe we can use a special `ButtonLayer` subclass, just like we already have a `ButtonControl`, and make it draw every cell of its super but reversed.

Reply:
So `ButtonControl` would override `makeLayer()` to return `ButtonLayer`, and `ButtonLayer` would override `cell(at:)`.